src/zkscript/bilinear_pairings/model/triple_miller_loop.py

```diff
@@ -153,7 +156,7 @@ def __one_step_without_addition(
             positive_modulo=positive_modulo,
             check_constant=False,
             clean_constant=(i == 0) and clean_constant,
-            verify_gradient=True,
+            verify_gradient=(verify_gradients >> 2) & 1,
```

Comment:
either if we decide to keep the bitmask or not, I would change this slightly. maybe we can have verify_gradient_1(2 and 3)? just to make it a bit more readable